flux_led/__init__.py

```diff
@@ -636,6 +638,10 @@ def setWarmWhite255(self, level, persist=True, retry=2):
         msg.append(0x00)
         msg.append(0x00)
         msg.append(int(level))
+
+        if self.protocol == "LEDENET":
```

Comment:
How about to make it case insensitive or forcing the string to be in lower case to avoid user errors?